<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_page</class>
 <widget class="QMainWindow" name="login_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>933</width>
    <height>641</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>934</width>
    <height>641</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget
{
background-image: url(:/new/prefix1/login_background.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#QMessageBox
{
background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);
}</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>931</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>378</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgba(255, 255, 255, 0.01);
border-image: url(:/new/prefix1/login_humannn.png);
color:rgb(0, 0, 80, 0.8);
</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>378</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_login">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>551</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>* login *</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(0, 0, 80, 0.1);
color :rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>  username :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(0, 0, 80, 0.1);
color :rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>  password :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_geetter">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kristen ITC</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(0, 98, 255);</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string> amin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass_getter">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kristen ITC</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(0, 98, 255);</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> 84#@pTk5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sign_up">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>264</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color :rgb(255, 255, 255);
background:rgb(0, 44, 127);</string>
     </property>
     <property name="text">
      <string>sign up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="guest">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>264</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color :rgb(255, 255, 255);
background:rgb(0, 44, 127);</string>
     </property>
     <property name="text">
      <string>guest</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="sign_in">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>264</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Old English Text MT</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color :rgb(255, 255, 255);
background:rgb(0, 44, 127);</string>
     </property>
     <property name="text">
      <string>sign in</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_top_of_log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 98, 0.5);
background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="refresh_btn_log">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>BusyCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 0.2);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="login_page_main.qrc">
       <normaloff>:/new/prefix1/closeee.svg</normaloff>:/new/prefix1/closeee.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn_log">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>WaitCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 0.2);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="login_page_main.qrc">
       <normaloff>:/new/prefix1/reloaddd.svg</normaloff>:/new/prefix1/reloaddd.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="time_jari">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 0.2);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_to_mainwindoww">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="login_page_main.qrc"/>
 </resources>
 <connections/>
</ui>
